### PR TITLE
Fix scrolling issues

### PR DESCRIPTION
**Pull request by sebastian-nehrdich**
These are some fixes to the text-view and for the display of the search results.

### CODE REVIEW

File: src/views/textview/text-view-right.js [hunk 1]
```diff
@@ -228,6 +227,7 @@ export class TextViewRight extends LitElement {
             activeSegment: segnr,
             position: position,
             selectedParallels: parallels,
+            limitCollection: [this.fileName],
```
Comment (ayya-vimala) [inline src/views/textview/text-view-right.js]: :+1:  - yes, that will fix a few things.Thanks!

File: src/views/textview/text-view.js [hunk 2]
```diff
@@ -338,7 +338,6 @@ export class TextView extends LitElement {
                   id="text-view-middle"
                   .score="${this.score}"
                   .fileName="${this.fileName}"
-                  .limitCollection="${this.limitCollection}"
```
Comment (ayya-vimala) [inline src/views/textview/text-view.js]: Is this correct? This is in the middle column, not the right column. Maybe it is not needed there ...

File: src/views/searchview/search-view-list-header.js [hunk 3]
```diff
@@ -6,7 +6,7 @@ import sharedStyles from '../data/data-view-shared.styles';
 @customElement('search-view-list-header')
 class SearchViewListHeader extends LitElement {
   @property({ type: String }) searchQuery;
-
+  @property({ type: String }) resultNumber;
   static get styles() {
```
Comment (ayya-vimala) [inline src/views/searchview/search-view-list-header.js]: Can you keep an empty line above each function? That will makes things much more readable.

File: src/views/textview/text-view-left.js [hunk 4]
```diff
@@ -108,13 +108,12 @@ export class TextViewLeft extends LitElement {
...
+      // this condition is met when we are endless scrolling downwards
       activeElement.scrollIntoView({ block: 'end', inline: 'nearest' });
-      this.scrollTop += 18;
     } else {
-      // this is the case when we are scrolling downwards
-      activeElement.scrollIntoView({ block: 'end', inline: 'nearest' });
-      this.scrollTop -= 18;
+      // this is the case when we are scrolling upwards
+      activeElement.scrollIntoView({ block: 'start', inline: 'nearest' });
+      //this.scrollTop -= 18;
```
Comment (ayya-vimala) [inline src/views/textview/text-view-left.js]: Should this comment be there?

File: src/views/textview/text-view.js (hunk 2 above)
Comment (sebastian-nehrdich) [reply src/views/textview/text-view.js]: You are right, its obsolete. Didn't notice...

Comment (sebastian-nehrdich) [reply src/views/textview/text-view.js]: ah wait! notice the -, this line is supposed to be deletd. that's right! that should happen!

File: src/views/textview/text-view-middle.js [hunk 5]
```diff
@@ -46,6 +44,7 @@ export class TextView extends LitElement {
   }
 
   async fetchMiddleParallels() {
+    console.log('FETCH MIDDLE PARALLELS DATA', this.data);
```
Comment (TrebuhD) [inline src/views/textview/text-view-middle.js]: we should probably delete this

**Pull request merged**